app/test/test_similarity_lang_analyzers.py

```diff
@@ -33,7 +33,7 @@ def test_all_analyzers(self):
         examples = [{ 'text': 'केले को कैसे काटें', 'language': 'hi'}, {'text': 'how to slice a banana', 'language': 'en'}, {'text': 'como rebanar un plátano', 'language': 'es'}, {'text': 'কিভাবে একটি কলা টুকরা করা হয়', 'language': 'bn'}]
         with self.client:
             for example in examples:
-                response = self.client.post('/text/similarity/', data=json.dumps(example), content_type='application/json')
+                response = self.client.post('/similarity/sync/text/', data=json.dumps(example), content_type='application/json')
```

Comment:
I think you probably need to add more structure to the `example` here - see something like `test_audio_basic_http_responses` in `test_sync_similarity.py` (and probably mock the response from Presto as we do there as well). Should just be a matter of adding a few more keys here:

```
{
    'text': 'केले को कैसे काटें',
    'language': 'hi',
    'project_media_id': 1,
    'context': {
        'team_id': 1,
    }
}
```

Reply:
We shouldn't have to mock presto here. We can avoid calls to Presto all together by leaving `models` off (as is) and/or setting it equal to `['elasticsearch']` so that we're not using any vector models and hence have no Presto calls.